docs/artefacts/offramps.md

```diff
@@ -73,6 +74,27 @@ The offramp `/offramp/system::stderr/system` can be used to print to STDERR. Dat
 ## Supported Offramps
 
 
+### amqp
+
+The amqp offramp publishes AMQP messages to an [AMQP](https://www.amqp.org) broker. It uses [lapin](https://docs.rs/lapin/1.6.8/lapin/) for an AMQP 0.9.1 protocol implementation.
+
+Example:
+
+```yaml
+offramp:
+  - id: amqp
+    type: amqp
+    config:
+      amqp_addr: "amqp://guest:guest@127.0.0.1:5672/"
+      exchange: ""
+      routing_key: "hello"
+      publish_options:
+        immediate: true
+        mandatory: false
+```
+
+Messages are published to the specified `exchange` (empty string is the default exchange) using the specified routing key. Lapin's [basic publish options](https://docs.rs/lapin/1.6.8/lapin/options/struct.BasicPublishOptions.html) are documented with the [protocol-spec for basic publish operation](https://www.rabbitmq.com/amqp-0-9-1-reference.html#class.basic).
```

Comment:
Could you create a small list of the available config options and concisely explain their data-type, format,meaning and optional/required/default value? Especially the `amqp_addr` is interesting here.

For the `publish_options` please linke to [mandatory](https://www.rabbitmq.com/amqp-0-9-1-reference.html#basic.publish.mandatory) and [immediate](https://www.rabbitmq.com/amqp-0-9-1-reference.html#basic.publish.immediate) and/or write something along those lines.

This would make it much easier for people to use your awesome offramp. :)